schema/schemahandler.go

```diff
@@ -403,6 +340,109 @@ func NewSchemaHandlerFromStruct(obj interface{}) (sh *SchemaHandler, err error)
 	return res, nil
 }
 
+func generateSchemaForSlice(item *Item, stack []*Item, schemaElements []*parquet.SchemaElement, infos []*common.Tag) ([]*Item, []*parquet.SchemaElement, []*common.Tag) {
```

Comment:
I don't see a dependency to `schemahandler.go` in `proto.go`. I assume it already exists? 😄

Reply:
They are in the same package, which doesn't require import.

Reply:
Got it, makes sense. Thanks!